### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by shdwy** [hop-protocol/hop-airdrop#456]
# Related Addresses

```
0x25c62baa5136df213ef5da05c923f1efc0917ec8
0x0e214d19f35f710226dc2ea06ca8064b4b3bb875
0x9b290005f8fd49a9336ce8352250580048959043
0x455183f35e540fff03d53c34904fee3d51eb8032
0x3179e38f3fc12e27272eb4fc8c33da4fb20677bf
0xc30063d3c07edccfec9c7d47be4fdd9f31417ed5
0x2ca33adbef6890df801ec06b0b4fda20699abb37
0x4a8cc61170723e24017e0c8d9b4b283d3d1c55a1
0x16b4b97a8347b12922c8a3d52264695eefe1752a
0xb52ac30460d3daa555be3c30a35eb11694b6aeca
0x3c4b42b0be4bca1e339d2b38f9196c5163a8c671
0x4b0d6c5192492e743431006326405ddacddfa913
```

# Reasoning

All addresses are originally funded from address 0xf6b94c41ba5dc73fb7a25dcfb3d623cbdfa57aaa with similar amounts of xDAI on the same day (most are done within seconds of each other). All addresses proceed to interact with the same couple of contracts right after being funded and then don't do anything else since. They all have 5-7 transactions ever.

# Methodology

I used blockscout API to figure out which address originally funded all 30k address eligible for the airdrop. Then I used excel to try and find groups of addresses being funded by the same EOA. I then analyzed the behaviour of groups of 10 or more addresses funded by one EOA to try and find similarities.

# Rewards Address

0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by shanefontaine** [comment]
Thank you for your report @shdwy. We have verified that the addresses in this report are Sybil attackers.

The report included 12 eligible addresses as Sybil attackers which means you are eligible for 2289.915860206003267511 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).


The qualified addresses are as follows:

```
0x25c62baa5136df213ef5da05c923f1efc0917ec8
0x0e214d19f35f710226dc2ea06ca8064b4b3bb875
0x9b290005f8fd49a9336ce8352250580048959043
0x455183f35e540fff03d53c34904fee3d51eb8032
0x3179e38f3fc12e27272eb4fc8c33da4fb20677bf
0xc30063d3c07edccfec9c7d47be4fdd9f31417ed5
0x2ca33adbef6890df801ec06b0b4fda20699abb37
0x4a8cc61170723e24017e0c8d9b4b283d3d1c55a1
0x16b4b97a8347b12922c8a3d52264695eefe1752a
0xb52ac30460d3daa555be3c30a35eb11694b6aeca
0x3c4b42b0be4bca1e339d2b38f9196c5163a8c671
0x4b0d6c5192492e743431006326405ddacddfa913
```

**Comment by shanefontaine** [comment]
In addition to the data you provided, here is a tabular view of their behavior on Hop protocol:

address | ensName | total | mainnet | arbitrum | optimism | polygon | xdai | totalVolume
-- | -- | -- | -- | -- | -- | -- | -- | --
0x455183f35e540fff03d53c34904fee3d51eb8032 |   | 6 | 0 | 0 | 0 | 2 | 4 | $2,656.92
0x4a8cc61170723e24017e0c8d9b4b283d3d1c55a1 |   | 6 | 0 | 0 | 0 | 3 | 3 | $2,568.57
0xc30063d3c07edccfec9c7d47be4fdd9f31417ed5 |   | 6 | 0 | 0 | 0 | 2 | 4 | $2,427.55
0x16b4b97a8347b12922c8a3d52264695eefe1752a |   | 5 | 0 | 0 | 0 | 2 | 3 | $2,081.21
0x3c4b42b0be4bca1e339d2b38f9196c5163a8c671 |   | 5 | 0 | 0 | 0 | 1 | 4 | $2,071.35
0xb52ac30460d3daa555be3c30a35eb11694b6aeca |   | 5 | 0 | 0 | 0 | 1 | 4 | $2,036.07
0x3179e38f3fc12e27272eb4fc8c33da4fb20677bf |   | 5 | 0 | 0 | 0 | 2 | 3 | $2,028.03
0x4b0d6c5192492e743431006326405ddacddfa913 |   | 5 | 0 | 0 | 0 | 1 | 4 | $2,021.38
0x2ca33adbef6890df801ec06b0b4fda20699abb37 |   | 5 | 0 | 0 | 0 | 3 | 2 | $1,960.09
0x0e214d19f35f710226dc2ea06ca8064b4b3bb875 |   | 4 | 0 | 0 | 0 | 2 | 2 | $1,812.95
0x25c62baa5136df213ef5da05c923f1efc0917ec8 |   | 4 | 0 | 0 | 0 | 1 | 3 | $1,524.91
0x9b290005f8fd49a9336ce8352250580048959043 |   | 3 | 0 | 0 | 0 | 1 | 2 | $1,516.36

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly? This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.
0xD47CB1835EB36e4A108C33E446449192426Fc160 This is a brand new empty address, why can't he provide the address he interacts with, because he is the real empty hunter. He circumvented it with better technical isolation. So here's my reasoning, I want 25% of his reward to go to me and then the remaining 75% to all eligible real users. Just like our rules.
@shanefontaine
